SU2_CFD/src/solvers/CFEASolver.cpp

```diff
@@ -1846,9 +1842,10 @@ void CFEASolver::BC_Sym_Plane(CGeometry *geometry, CNumerics *numerics, const CC
 
     /*--- Set and enforce 0 solution for mesh deformation ---*/
     nodes->SetBound_Disp(iPoint, axis, 0.0);
-
     LinSysSol(iPoint, axis) = 0.0;
-    LinSysReact(iPoint, axis) = 0.0;
+    if (config->GetMarker_All_Deform_Mesh_Sym_Plane(val_marker) == NO){
+      LinSysReact(iPoint, axis) = 0.0;
+    }
```

Comment:
I think there might be a more elegant way to do this check, I'll get back to you.

Reply:
I thought there was a boolean but there is not, you can use
`if (LinSysReact.GetLocSize() > 0) ...`
This way we save users some debugging if they try to use that marker type with the FEA solver.

Reply:
Great! Thank you very much, I really appreciate your help